packet/utils_for_test.go

```diff
@@ -11,38 +11,25 @@ import (
 	"github.com/intel-go/yanff/low"
 )
 
-// testMempool is common for all tests
-var testMempool *low.Mempool
+// isInit is common for all tests
+var isInit bool
 
 const (
 	payloadSize = 100
 )
 
-func init() {
-	if testMempool != nil {
-		GetMempoolForTest()
-	}
-}
-
-// GetMempoolForTest initialize DPDK and testMempool.
-// Calling this function guarantee initialization is done once during all tests in package.
-func GetMempoolForTest() *low.Mempool {
-	if testMempool == nil {
+func TInitDPDK() {
```

Comment:
Comment to exported function is required by golint. Maybe better Test instead of T?

Reply:
Test__ is a function for testing? it should have different signature

Reply:
This shouldn't be exported. As well as GetPacket. Will correct this.

Reply:
It is not obvious, but it is ok to export in this file
Directive // +build !test 
makes exported functions present only if build for testing